### PR TITLE
Add Synapse Integration test, Update Github CI

### PR DESCRIPTION
**Pull request by wpoehlm**
- Addition of main2.cwl, which tests the functionality of testing using synapse downloads
- Updated deprecated command in the Github CI

### CODE REVIEW

File: subworkflows/echo-workflow.cwl
Comment (tthyer) [inline subworkflows/echo-workflow.cwl:16]: This should always use a version number for the sake of stability.

**Review by tthyer — CHANGES_REQUESTED** [review]
@wpoehlm I don't mind adding this; I had previously created the integration test example in a branch, `ci_integration_test_example`, as indicated in the README, but I see how that could be easy to miss. Please take a look at whether there are any significant differences between your changes and my branch; if not, please feel free to merge what you've done. Please also update the README -- we should remove my branch and indicate that the new cwl exists to demonstrate the integration test.

File: subworkflows/echo-workflow.cwl
Comment (wpoehlm) [reply subworkflows/echo-workflow.cwl]: good call, fixed

**Comment by BrunoGrandePhD** [comment]
My hands are full with urgent work, so I defer to @tthyer on this. If you insist on getting my review, just request it, but it'll have to wait until later next week.

**Comment by wpoehlm** [comment]
> 
> 
> @wpoehlm I don't mind adding this; I had previously created the integration test example in a branch, `ci_integration_test_example`, as indicated in the README, but I see how that could be easy to miss. Please take a look at whether there are any significant differences between your changes and my branch; if not, please feel free to merge what you've done. Please also update the README -- we should remove my branch and indicate that the new cwl exists to demonstrate the integration test.

Sorry for the confusion, I used your ci_integration_test_example branch as the starting point, just didn't want to work directly on someone else's branch. Will delete that branch after merging here

**Review by tthyer — CHANGES_REQUESTED** [review]
This looks good, but please go ahead and remove the branch and update the README.

**Review by tthyer — APPROVED** [review]

**Pull request merged**
